html/about.html

```diff
@@ -0,0 +1,60 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="description" content="This is a web site for newbie amateur astronomers." />
+    <title>What Can I See Tonight?</title>
+
+    <link rel="stylesheet" href="../css/reset.css" type="text/css" />
+    <link rel="stylesheet" href="../css/styles.css" type="text/css" />
+    <link href='http://fonts.googleapis.com/css?family=Roboto:400,400italic,700' rel='stylesheet' type='text/css'>
+  </head>
+  <body>
+      <div class="container clearfix">
```

Comment:
if that 600px width was taking effect, this would prevent your page's backgrounds from stretching full width. you'd want to use the container class only inside another element that can reach full width